src/substrait/textplan/ParseResult.h

```diff
@@ -18,42 +18,41 @@ class ParseResult {
       SymbolTable symbolTable,
       std::vector<std::string> syntaxErrors,
       std::vector<std::string> semanticErrors) {
-    symbol_table_ = std::make_shared<SymbolTable>(std::move(symbolTable));
```

Comment:
Class data members in Google C++ style use underscores.  Have we documented the style Substrait C++ that I missed?

Reply:
Even the google source code doesn't follow the google c++ style strictly , see  [this](https://github.com/googleapis/google-cloud-cpp/blob/main/.clang-tidy)